pyw3d/blender_actions/color.py

```diff
@@ -33,11 +33,13 @@ def start_string(self):
         script_text = []
         script_text.extend([
             "new_color = {}".format(self.color),
+            "W3D_LOG.debug('{}')".format(self.color),
```

Comment:
It would probably be better to include more descriptive debugging output here, or just drop this logging statement before pulling this into develop.